builds/e2e/templates/unlock-test-agents.yaml

```diff
@@ -31,10 +31,10 @@ steps:
         --request PUT "https://msazure.visualstudio.com/_apis/distributedtask/pools/$POOL_ID/agents/$agentId/usercapabilities" \
         -H "Content-Type:application/json" \
         -H "Accept: application/json;api-version=5.0;" \
-        --max-time 10 \
-        --retry 5 \
+        --max-time 15 \
+        --retry 10 \
         --retry-delay 0 \
-        --retry-max-time 40 \
+        --retry-max-time 80 \
```

Comment:
I think we should also echo the $responseCapabilities and maybe the return code as well to better understand what is the output when the failure occurs

Reply:
Yeah I can do that. Will make the change.

Reply:
Fixed. Will wait before build finishes before merging:
https://dev.azure.com/msazure/One/_build/results?buildId=52822500&view=logs&j=87e23809-25cb-56fc-f202-d7fa2974c153&t=8e68d342-8c11-5a73-086a-b96933288846